packs/alertmanager/templates/alertmanager.nomad.tpl

```diff
@@ -0,0 +1,74 @@
+job [[ template "job_name" . ]] {
+  [[ template "region" . ]]
+  datacenters = [[ .alertmanager.datacenters | toPrettyJson ]]
+
+  [[ if .alertmanager.constraints ]][[ range $idx, $constraint := .alertmanager.constraints ]]
+  constraint {
+    attribute = [[ $constraint.attribute | quote ]]
+    value     = [[ $constraint.value | quote ]]
+    [[- if ne $constraint.operator "" ]]
+    operator  = [[ $constraint.operator | quote ]]
+    [[- end ]]
+  }
+  [[- end ]][[- end ]]
+
+  group "alertmanager" {
+    count = [[ .alertmanager.count ]]
+
+    network {
+      mode = "bridge"
```

Comment:
Can we make this configurable, so that the job can be deployed to non-linux clients? The Prometheus node exporter pack has an [example of this style](https://github.com/hashicorp/nomad-pack-community-registry/blob/main/packs/prometheus_node_exporter/variables.hcl#L41-L53).

Reply:
updated as requested